package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/applicationinsights-angularplugin-js",
-  "version": "14.0.0",
+  "version": "14.0.1",
```

Comment:
Please update the `publishConfig` to say `Angular14` (we already use `Angular11` for the previous release, so this would be inline with that naming.